File: corpus/go/fixture_14.go
package fixture14

type PriceBook14 struct {
    Multiplier float64
}

func CalculateDiscount14(cartItems []float64, taxRate float64) float64 {
    totalAmount := 0.0
    for _, price := range cartItems {
        totalAmount += price
    }
    if totalAmount > 140.0 {
        return totalAmount * (1.0 + taxRate)
    }
    return totalAmount
}

File: corpus/javascript/fixture_14.js
export function calculateDiscount14(cartItems, taxRate) {
  const totalAmount = cartItems.reduce((sum, item) => sum + item.price * (item.qty ?? 1), 0);
  return totalAmount > 140 ? totalAmount * (1 + taxRate) : totalAmount;
}

export class PriceBook14 {
  lookupPrice(sku) { return sku.length + 14; }
}
